compiler/extensions/cpp/runtime/src/zserio/ZserioTreeCreator.h

```diff
@@ -5,6 +5,7 @@
 #include <type_traits>
 #include <cerrno>
 #include <cstdlib>
+#include <cstdint>
```

Comment:
Zserio uses Types.h to allow to override the default standard types header. I'm not sure if anybody is still using it, but it was requested years ago to support some obscure platform....

Reply:
Should `<cstdint>` added to `Types.h` then? `Types.h` already includes `"stdint.h"` which defines defines the same types but outsides the `std::` namespace.

Reply:
`stdint.h` was already exchanged for `cstdint` on `clang-tidy-ci` branch ... but I'm know thinking if it's assured that the types will be defined in global namespace - they are on all tested compilers.

Can you please check if it's enough to have only `#include <cstdint>` on GCC 13.0.1?
Seems to work: https://godbolt.org/z/Mvxqrox6z

Reply:
I just noticed that we use `using ZserioTreeCreator = BasicZserioTreeCreator<std::allocator<std::uint8_t>>;` with `std::uint8_t`, but it was unintentional since we don't use the integral types via std namespace in zserio runtime. Isn't it enough just change it to global `uint8_t`?
Can you please confirm that the error was on this line?

Reply:
Yes, that was the problem. Thanks.